sdk/storage/Azure.Storage.Files/src/Models/FileSmbProperties.cs

```diff
@@ -154,13 +156,53 @@ public bool Equals(FileSmbProperties other)
             && this.FileId == other.FileId
             && this.ParentId == other.ParentId;
 
+        /// <summary>
+        /// ToString
+        /// </summary>
+        /// <returns>string</returns>
+        public override string ToString()
+        {
+            var stringBuilder = new StringBuilder();
+            stringBuilder.AppendLine(nameof(FileSmbProperties) + ":");
+            stringBuilder.AppendLine(nameof(this.FileAttributes) + ": " + this.FileAttributes.ToString());
```

Comment:
Does `FileAttributes` have a `ToString`?  I'm fine removing all these `ToString` calls, btw.  I'm not sure it's particularly helpful to log the `FileSmbProperties`.

Should we indent the property values in this method?

Reply:
Would it be ok if we not log FileSmbProperties and remove it's ToString method?

Reply:
Yes - I think that sounds good.

Reply:
fixed.